### ISSUE TITLE
Single Teams data extract

### ISSUE THREAD

**Issue by hermannatorii** [For-The-Wolf/zwift-scrape#7]
Is it possible to extract only a single teams data?

Cheers

**Comment by For-The-Wolf** [comment]
My memory is that teams are included in the same table cell as the username of the rider. It should be possible to filter the results to only save rows with the given team's tag present.

Happy to add this as a option later this week or feel free to try and do it yourself in the meantime

**Comment by hermannatorii** [comment]
Yeah, I am in a new role at my workplace where I am getting involved with coding including C and Python. So would be nice to learn
If you could add this as an option it would be so well appreciated. would be nice to see how you do it so I can learn and do my own work!

Thanks so much for reply and all your hard work

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Comment by hermannatorii** [comment]
I still havent fathemed this out!
Could you help me or give me some pointers?
trying to learn about coding still
